libraries/package-extractor/src/PackageExtractor.ts

```diff
@@ -120,6 +124,34 @@ export interface IExtractorProjectConfiguration {
   dependenciesToExclude?: string[];
 }
 
+/**
+ * The extractor configuration for individual dependencies.
+ *
+ * @public
+ */
+export interface IExtractorDependencyConfiguration {
+  /**
+   * The name of dependency
+   */
+  dependencyName: string;
+  /**
+   * The version of dependency
+   */
+  dependencyVersion: string;
+  /**
+   * A list of glob patterns to exclude when extracting this dependency. If a path is
+   * matched by both "patternsToInclude" and "patternsToExclude", the path will be
+   * excluded. If undefined, no paths will be excluded.
+   */
+  patternsToExclude?: string[];
```

Comment:
Are these paths? Maybe update the name to something like `filePathPatternsToExclude` and `filePathPatternsToInclude` to be completely clear.

Reply:
They are glob patterns, same as option filed in projectConfigurations.